neqo-http3/src/connection.rs

```diff
@@ -916,39 +910,22 @@ impl Http3Connection {
             return Err(Error::UnexpectedFrame);
         } else {
             // Issue reset events for streams >= goaway stream id
-            self.transactions_client
+            for id in self
+                .transactions_client
                 .iter()
                 .filter(|(id, _)| **id >= goaway_stream_id)
                 .map(|(id, _)| *id)
```

Comment:
want to use filter_map() ?

Reply:
it's no shorter because you have to use a conditional to return Some(*id) or None, so I'm going to leave it.